src/test/java/com/Searchutil/app/SearchServiceUtilTest.java

```diff
@@ -0,0 +1,49 @@
+package com.Searchutil.app;
```

Comment:
Refactor package structure here also, try to map it with what we have in _src/main_